regions/_geometry/pnpoly.pxd

```diff
@@ -0,0 +1,5 @@
+cimport numpy as np
```

Comment:
Add standard license line at the top, even if it's a very small file.

Can you add a docstring with a sentence saying why we need a `pxd` file at all?
(my Cython is always rusty because I use it so rarely. It's the same for most Python programmers.)